catalog/dags/common/loader/provider_details.py

```diff
@@ -124,7 +124,13 @@
 
 # User-Agent header for APIs that require it
 CONTACT_EMAIL = os.getenv("CONTACT_EMAIL")
-UA_STRING = f"Openverse/0.1 (https://wordpress.org/openverse; {CONTACT_EMAIL})"
+
+CANONICAL_DOMAIN: str = os.getenv("CANONICAL_DOMAIN", "openverse.org")
+
+_proto = "http" if "localhost" in CANONICAL_DOMAIN else "https"
+CANONICAL_ORIGIN: str = f"{_proto}://{CANONICAL_DOMAIN}"
```

Comment:
Is this just for consistency with the api? Would we ever actually use a domain with "localhost" in this context?

Reply:
Yes, this arose from my conversation [with Sara here](https://github.com/WordPress/openverse/pull/3828#discussion_r1503265924). We opted for keeping consistency with the API in any case.